NativeApp/src/components/Login/Form/index.js

```diff
@@ -38,45 +38,68 @@ class LoginForm extends Component {
-        />
-        <TextInput
-          id="input-2"
-          onFocus={() => changeColor('input-2', '#00DCFA')}
-          onBlur={() => changeColor('input-2', 'gray')}
-          placeholder="Password"
-          onChangeText={text => this.setState({ password: text })}
-          value={password}
-          secureTextEntry
-          underlineColorAndroid="transparent"
-          style={{ borderBottomWidth: 1, borderColor: hasError ? 'red' : underlineColor2 }}
-          selectionColor="#00DCFA"
-        />
-        {
-          hasError && (
-            <Caption type="base" style={styles.red}>
-              Incorrect email or password
-            </Caption>
-          )
-        }
+      <View style={styles.view}>
+        <View>
+          <FormLabel>Email</FormLabel>
+          <FormInput
+            id="input-1"
+            onChangeText={text => this.setState({ email: text })}
+            inputStyle={{
+              fontSize: 20,
+              color: 'black',
+              padding: 8,
+              borderColor: hasError ? 'red' : underlineColor1,
+              borderBottomWidth: 1,
+              width: '100%',
+            }}
+            underlineColorAndroid="transparent"
+            selectionColor="#00DCFA"
+            placeholder="you@email.com"
+            autoCapitalize="none"
+            value={email}
+            returnKeyType="next"
+            onSubmitEditing={() => this.secondInput.focus()}
+            onFocus={() => changeColor('input-1', '#00DCFA')}
+            onBlur={() => changeColor('input-1', 'gray')}
+            blurOnSubmit={false}
+          />
+          <FormLabel>Password</FormLabel>
+          <FormInput
+            id="input-2"
+            textInputRef={(el) => { this.secondInput = el; }}
+            onChangeText={text => this.setState({ password: text })}
+            inputStyle={{
+              fontSize: 20,
+              color: 'black',
+              padding: 8,
+              borderColor: hasError ? 'red' : underlineColor2,
+              borderBottomWidth: 1,
+              width: '100%',
+            }}
+            underlineColorAndroid="transparent"
+            selectionColor="#00DCFA"
+            placeholder="*******"
+            autoCapitalize="none"
+            value={password}
+            secureTextEntry
+            returnKeyType="done"
+            onFocus={() => changeColor('input-2', '#00DCFA')}
+            onBlur={() => changeColor('input-2', 'gray')}
+            onSubmitEditing={() => handleLogin(email, password)}
+          />
+          {
+            hasError && (
+              <FormValidationMessage>Incorrect email or password</FormValidationMessage>
+            )
+          }
+        </View>
         <Button
           onPress={() => handleLogin(email, password)}
-          title="Login"
+          title="Log In"
           textStyle={{ fontWeight: 'bold', color: '#fff', fontSize: 20 }}
```

Comment:
Styles into StyleSheets below and use global text styles.

Reply:
done :)